app/views/events/index.html.erb

```diff
@@ -17,6 +17,7 @@
           <td><%= event.duration %></td>
           <td><%= event.started_at %></td>
           <td><%= link_to 'Show', event %></td>
+          <td><%= link_to 'Add me to this event', add_me_event_path(event) %></td>
```

Comment:
Please always add translations to new links. You can also fix translations for other links.